lib/arel/visitors/clickhouse.rb

```diff
@@ -64,6 +65,11 @@ def visit_Arel_Nodes_Using o, collector
         collector
       end
 
+      def visit_Arel_Nodes_LimitBy(o, collector)
+        collector << "LIMIT #{o.expr} BY #{o.column}"
```

Comment:
The o.column should be sanitized to avoid SQL injection. And maybe we should ensure that the limit is a valid positive number

Reply:
there's a `sanitize_limit` method in `DatabaseStatements` class. (Limits are not added with visitors it seems)

Reply:
I added some basic validation for the node
From what I see, there is no win/win method to validates that any other clause are present.
This is also not done in any other nodes or clause addings